os/binfmt/libelf/libelf_init.c

```diff
@@ -181,6 +185,19 @@ int elf_init(FAR const char *filename, FAR struct elf_loadinfo_s *loadinfo)
 		return ret;
 	}
 
+	if (loadinfo->compression_type > 0) {
+#ifdef CONFIG_COMPRESSED_BINARY
+		ret = compress_init(loadinfo->filfd, loadinfo->offset, &loadinfo->filelen);
+		if (ret != OK) {
+			berr("Failed to read header for compressed binary : %d\n", ret);
+			return ret;
+		}
+#else
+		berr("No support for reading compressed binary\n");
```

Comment:
We can select CONFIG_COMPRESSION_TYPE to 0. In this case, load will be failed.
What is our rule for loading? If only compressed type, selection 0 is not needed.

Reply:
Made this change in os/kernel/Kconfig.
CONFIG_COMPRESSION_TYPE cannot be 0 now